### DISCUSSION TITLE
super loop

### DISCUSSION THREAD

**Discussion by Fasteroid** [wiremod/advdupe2#404] [General]
https://github.com/wiremod/advdupe2/blob/b83f6fd028b4c7bc3d2cf54394376a120e81948c/lua/weapons/gmod_tool/stools/advdupe2.lua#L47-L69

dear god what the hell is this

**Comment by Divran** [comment]
a super loop

**Comment by thegrb93** [comment]
It's a brute force constraint system sorter iirc.

**Reply by Rjason92** [reply]
Hey sup what my error
